projects/planner/src/app/diff/waypoints/diff-waypoints.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="card overflow-y-scroll">
+    <div class="card-header">
+        <div class="float-start">
+            <fa-icon [icon]="tasks" class="me-3"></fa-icon>
+            <span>Dive way points</span>
+        </div>
+    </div>
+    <div class="card-body card-minheight table-responsive card-max-height-md">
+        <table id="waypointsDiffTable" class="table table-sm p-0 text-center table-hover">
+            <thead class="table-light sticky-top">
+            <tr>
+                <th colspan="1" ></th>
+                <th colspan="2" class="fw-bold px-1 active-profile">Profile A</th>
+                <th colspan="2" class="fw-bold px-1">Profile B</th>
+            </tr>
+            <tr>
+                <th class="fw-bold px-1 ">Run [min]</th>
+                <th class="fw-bold px-1 active-profile">Depth [{{units.length}}]</th>
+                <th class="fw-bold px-1 active-profile">Duration [min]</th>
+                <th class="fw-bold px-1">Depth [{{units.length}}]</th>
+                <th class="fw-bold px-1">Duration [min]</th>
+            </tr>
+            </thead>
+            <tbody>
+                <tr *ngFor="let row of waypointRows">
+                    <td class="px-1 vertical-right-border">
+                        {{ row.runTime * 1000 | date:'mm:ss' }}
+                    </td>
+                    <td  *ngIf="row.depthA !== undefined; else emptyRow" class="px-1">
+                        {{ row.depthA  | number: '1.0-0'}}
+                    </td>
+                    <td *ngIf="row.durationA !== undefined else emptyRowWithDivider" class="px-1 vertical-right-border">
+                        {{ row.durationA * 1000 | date: 'mm:ss'}}
```

Comment:
We have formatting based on the value, see original component

Reply:
Will fix once importing profiles into DiveSchedules is implemented